### PR TITLE
Setup CD

### PR DESCRIPTION
**Pull request by codeAesthetic**

### CODE REVIEW

**Comment by codeAesthetic** [comment]
forked from #9

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -35,12 +35,8 @@ jobs:
...
+        
       - name: Create build
         run: yarn run build
-
-      - name: Check for test coverage
-      - uses: artiomtr/jest-coverage-report-action@v1.3
-        with:
-            github_token: ${{ secrets.GITHUB_TOKEN }}
-            test_script: yarn test:coverage
-            threshold: 70
```
Comment (chirgjn) [inline .github/workflows/integration.yml:46]: Can you put these changes in a separate PR so that we can revert them later.

**Review by chirgjn — CHANGES_REQUESTED** [review]

File: .github/workflows/integration.yml (hunk 1 above)
Comment (codeAesthetic) [reply .github/workflows/integration.yml:46]: We have to revert it now otherwise we will get a failed status check on this PR.

Comment (chirgjn) [reply .github/workflows/integration.yml:46]: No we will submit the other (coverage) PR first

**Review by chirgjn — APPROVED** [review]
LGTM

**Pull request merged**